core/dbt/task/runnable.py

```diff
@@ -414,6 +416,10 @@ def execute_nodes(self):
 
         return self.node_results
 
+    @staticmethod
+    def _pool_thread_initializer(ci):
```

Comment:
nit: is `ci` the InvocationContext? If so, we can name it `ic` and type annotate for clarity

Reply:
oops, yes